### DISCUSSION TITLE
How to fit bounds automatically in order to see all the markers

### DISCUSSION THREAD

**Discussion by Amaurgit** [NathanAP/vue-google-maps-community-fork#49] [Q&A] [answered]
Hey ! Nice work here :)

Did someone managed to implements the fitbounds feature ? I'm trying to make my map auto zoom to where I can see all my markers.

Any idea? 

Thanks !!

**Comment by tahaipek** [comment, marked as answer]
Can you try this solution?

```
 data() {
    return {
      markers: [],
    };
  },
  methods: {

    fitBoundsOnMap() {
      this.$refs.googleMapRef.$mapPromise.then((mapObject) => {

        var bounds = new google.maps.LatLngBounds();

        this.markers.forEach((marker) => {
          bounds.extend(marker.position);
        });

        mapObject.fitBounds(bounds);
      });
    },
},

```

**Reply by felixranesberger** [reply]
I think we should reference this solution inside the readme, as this should be a problem, that many people face.

**Reply by felixranesberger** [reply]
I created a pull request: https://github.com/NathanAP/vue-google-maps-community-fork/pull/76
